examples/gno.land/p/demo/simpledao/dao.gno

```diff
@@ -0,0 +1,202 @@
+package simpledao
+
+import (
+	"errors"
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/gov/dao"
+)
+
+var (
+	ErrInvalidExecutor           = errors.New("invalid executor provided")
+	ErrInsufficientProposalFunds = errors.New("insufficient funds for proposal")
+	ErrInsufficientExecuteFunds  = errors.New("insufficient funds for executing proposal")
+	ErrProposalExecuted          = errors.New("proposal already executed")
+	ErrProposalInactive          = errors.New("proposal is inactive")
+	ErrProposalNotAccepted       = errors.New("proposal is not accepted")
+	ErrExecutorExpired           = errors.New("executor is expired")
+)
+
+var (
+	minProposalFeeValue int64 = 100 * 1_000_000 // minimum gnot required for a govdao proposal (100 GNOT)
+	minExecuteFeeValue  int64 = 500 * 1_000_000 // minimum gnot required for a govdao proposal (500 GNOT)
+
+	minProposalFee = std.NewCoin("ugnot", minProposalFeeValue)
+	minExecuteFee  = std.NewCoin("ugnot", minExecuteFeeValue)
+)
+
+// SimpleDAO is a simple DAO implementation
+type SimpleDAO struct {
```

Comment:
SimpleDAO is effectively the implementation that:
- Fulfills the DAO interface
- Manages members (from the member store package which should handle 99% of the logic)
- Manages voting (no need to match an interface, but I think that @ajnavarro made something smart that can be recycled here, now, or later)